example/relational/README.md

```diff
@@ -178,7 +180,14 @@ Composite sort key supports hierarchical one-to-many relations. Additional ortho
 * As a reader I want to list all articles written by the author ...
 * As a reader I want to look up all articles for a given category in chronological order ...
 
-The first access pattern is addressed by composite sort key `author ⟼ article`, the second one requires another `category ⟼ year` key. One approach is an explicit projection of data but secondary indexes are easy. DynamoDB implicitly copies data from the main table into the secondary index using another pair of attributes to establish identity. Therefore a new access dimension is unlocked. Eventual consistency is only the aspect to consider while using indexes. The local secondary indexes provide strong consistency but general advice to favour global indexes.
+The first access pattern is addressed by composite sort key `author ⟼ article`,
+=> this is a bit disturbing to me ^^' isn't composite partition key?, is it the same? 
+=> do I understand correctly that a sort key cannot be accessed directly without partiton key?
+
+the second one requires another `category ⟼ year` key. One approach is an explicit projection of data but secondary indexes are easy. 
+=> `but secondary indexes are easy` ?
+DynamoDB implicitly copies data from the main table into the secondary index using another pair of attributes to establish identity. Therefore a new access dimension is unlocked. Eventual consistency is only the aspect to consider while using indexes. The local secondary indexes provide strong consistency but general advice to favour global indexes.
+=> Do I understand that actually local secondary index would be problematic for this use case as the partition key remains the same with the main table? 
```

Comment:
Yes `category ⟼ year` is not possible to implement with local indexes. You can build `author  ⟼ year` local index. Then, also please check limitation about local index. They are not flexible enough.